crud/stats/local_registry.lua

```diff
@@ -0,0 +1,124 @@
+local dev_checks = require('crud.common.dev_checks')
+local registry_common = require('crud.stats.registry_common')
+
+local registry = {}
+local internal_registry = {}
+
+--- Initialize local metrics registry
+--
+--  Registries are not meant to used explicitly
+--  by users, init is not guaranteed to be idempotent.
+--
+-- @function init
+--
+-- @treturn boolean Returns true.
+--
+function registry.init()
+    internal_registry.spaces = {}
+    internal_registry.space_not_found = 0
+
+    return true
+end
+
+--- Destroy local metrics registry
+--
+--  Registries are not meant to used explicitly
+--  by users, destroy is not guaranteed to be idempotent.
+--
+-- @function destroy
+--
+-- @treturn boolean Returns true.
+--
+function registry.destroy()
+    internal_registry = {}
+
+    return true
+end
+
+--- Get copy of local metrics registry
+--
+--  Registries are not meant to used explicitly
+--  by users, get is not guaranteed to work without init.
+--
+-- @function get
+--
+-- @tparam string space_name
```

Comment:
Thanks for the link (and commit with build instructions). Since you've opened https://github.com/tarantool/crud/pull/254 , I remove this commit from this PR to not confuse any other reviewers.

I've tried to build this doc with https://github.com/tarantool/crud/pull/254 and fixed all issues that I found.